synapse/app/homeserver.py

```diff
@@ -182,6 +183,14 @@ def _configure_named_resource(self, name, compress=False):
                 "/_matrix/client/versions": client_resource,
             })
 
+        if name == "consent":
+            consent_resource = ConsentResource(self)
+            if compress:
+                consent_resource = gz_wrap(consent_resource)
+            resources.update({
+                "/_matrix/consent": consent_resource,
```

Comment:
If this isn't a matrix API then we should probably host it under e.g. `/_synapse` or wherever we host synapse specific stuff.

Reply:
that would be /matrix/client/r0/admin, mostly...

Reply:
"oh"